packaging/fedora/snapd.spec

```diff
@@ -803,6 +803,9 @@ popd
 %sysctl_apply 99-snap.conf
 %endif
 %systemd_post %{snappy_svcs}
+%if !(0%{?rhel} && 0%{?rhel} < 8)
+%systemd_user_post %{snappy_user_svcs}
```

Comment:
Why is this not being executed on EL7? The macro exists there...

Reply:
I was getting some failures on one of the RHEL7 derivatives (I think Amazon Linux 2?) earlier in this branch's development, and adding this conditional seemed to fix it.  I've just removed the conditionals and all the tests are passing so I'm not sure what was going on there.

I guess I jumped to the conclusion that the user instance of systemd was non-functional since it didn't seem to be running when I booted up the CentOS 7 live GNOME image.